spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/abstractnn/AbstractModule.scala

```diff
@@ -354,6 +354,9 @@ abstract class AbstractModule[A <: Activity: ClassTag, B <: Activity: ClassTag,
    */
   def parameters(): (Array[Tensor[T]], Array[Tensor[T]]) = null
 
+
+  def setParameters(params : Array[Tensor[T]]): Unit = {}
```

Comment:
private[bigdl]?

Reply:
should we group the parameter relate API to a trait? As the AbstractModule is too large now...